### DISCUSSION TITLE
Does Kernel 5.15.50 support EMMC boot on s905w2 box?

### DISCUSSION THREAD

**Discussion by iisnsb** [devmfc/debian-on-amlogic#115] [Q&A] [answered]
First of all,you do a good job.I thank you very much because of your work makes my box can boot ubuntu.But I found some problems that I couldn’t use  edltool https://github.com/bkerler/edl on my box.I always got many errors when I installed  edl.
I need Ubuntu 22.04 LTS.But I am worried about something what I can’t boot it on emmc .

**Comment by devmfc** [comment]
You're welcome.

No, kernel 5.15.50 is not able to boot on s905w2 (S4) amlogic SoC's
Is Ubuntu 22.04 really the only release edl can work on?!!? That's kind'a strange...

You could use a newer kernel in an older image though..

**Reply by devmfc** [reply, marked as answer]
I've created a one-off image with newer kernel and Ubuntu Jammy:

[Ubuntu 22.04 LTS Jammy with kernel 6.12.6](https://github.com/devmfc/unofficial/releases/tag/Jammy6.12)

But I doubt it will solve your problems though, good luck

**Reply by iisnsb** [reply]
Thanks

**Comment by iisnsb** [comment]
Thank you sir
